Starting dlv on a process immediately segfaults

Hi, 
We are compiling our code via go and try to debug it via dlv.
When we executing the next line : 
`dlv exec <binary> `
We get a segfault when delve is trying to read the ELF contents.
Here is the stacktrace:
```
API server listening at: [::]:36693
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x69706e]

goroutine 11 [running]:
github.com/derekparker/delve/pkg/proc.(*BinaryInfo).setGStructOffsetElf(0xc42009b600, 0xc4201240a0, 0xc42011e380)
                /usr/local/go/src/github.com/derekparker/delve/pkg/proc/bininfo.go:675 +0x14e
created by github.com/derekparker/delve/pkg/proc.(*BinaryInfo).LoadBinaryInfoElf
                /usr/local/go/src/github.com/derekparker/delve/pkg/proc/bininfo.go:619 +0x31f
```
The binary otherwise runs perfectly outside of delve.

1. What version of Delve are you using (`dlv version`)?
   v1.1.0 release
2. What version of Go are you using? (`go version`)?
    go1.8.7
3. What operating system and processor architecture are you using?
   Linux linux-vta3 4.20.0-1-default #1 SMP PREEMPT Tue Jan 8 08:38:29 UTC 2019 (34f009a) x86_64 x86_64 x86_64 GNU/Linux
4. What did you do?
    Execute the binary file via delve (also fails when attaching to the process)
5. What did you expect to see?
     Delve running out process
6. What did you see instead?
        SIGSEGV

--- a/pkg/proc/bininfo.go
+++ b/pkg/proc/bininfo.go
@@ -756,6 +756,10 @@ func (bi *BinaryInfo) setGStructOffsetElf(exe *elf.File, wg *sync.WaitGroup) {
 			break
 		}
 	}
+	if tls == nil {
+		bi.gStructOffset = ^uint64(8) + 1 // -8
+		return
+	}
 	memsz := tls.Memsz
 
 	memsz = (memsz + uint64(bi.Arch.PtrSize()) - 1) & ^uint64(bi.Arch.PtrSize()-1) // align to pointer-sized-boundary

